bin/mantle-dynamics-L0/0_overview_mantle_dynamics_L0.ipynb

```diff
@@ -0,0 +1 @@
e":"markdown","id":"4fafd930-00d8-4fad-8a64-5ade7bd97a40","metadata":{"tags":[],"id":"4fafd930-00d8-4fad-8a64-5ade7bd97a40"},"source":["## Description\n","\n","This module uses a community software ASPECT to describe the dominant mechanisms of heat transport inside the Earth's mantle—conduction and convection.\n","\n","## Learning Objectives\n","\n","###  Scientific\n","Upon completion of this module, you should be familiar with:\n","  * Driving forces of the plates\n","  * How the values of dimensionless numbers—Rayleigh number and Nusselt number—govern convective and conductive heat transport mechanisms in the Earth's mantle.\n","  * Different tectonic regimes\n","  * Mantle plumes\n","\n","### Technical\n","Upon completion of this module you should:\n","  * be able to run ASPECT from the command line,\n","  * understand the basic layout of model simulation file,\n","  * be able to visualize simulation results\n","\n","## Prerequisites\n","\n","  * Installed ASPECT (> version 2.2),\n","  * Basic knowledge of python modules (matplotlib, pandas, scipy) for scientific computing and analyis\n","  * Fundamentals of heat transport mechanisms inside Earth\n"]},{"cell_type":"markdown","id":"8c42746c","metadata":{"id":"8c42746c"},"source":["## Table of Contents\n"," * [ASPECT philosophy](./1_ASPECT_philosophy.ipynb)\n","   * [Understanding the paramter file](./1-1_understanding_the_parameter_file.ipynb)\n"," * [Plate tectonics and basic equations](./2_plate_tectonics_and_basic_equations.ipynb)\n"," * [Convection in a box model](./3_convection_in_box_model.ipynb)\n"," * [Rayleigh Nusselt Relationship](./4_Rayleigh_Nusselt_Relationship.ipynb)\n"," * [Different tectonic regimes](./5_different_tectonic_regimes.ipynb)\n"]},{"cell_type":"markdown","id":"61b819f4","metadata":{"id":"61b819f4"},"source":["***\n","## Acknowledgements\n","\n","Course materials are based on the tectonics modeling workshop 2020 led by ASPECT developers—Wolfgang Bangerth,\n","Juliane Dannberg, Menno Fraters, Rene Gassmoeller, Anne Glerum, Timo Heister, Bob Myhill, John Naliboff, and\n","Cedric Thieulot. The slides and the material are available in the [github repository](https://github.com/gassmoeller/aspect-tutorials)."]},{"cell_type":"markdown","id":"7514e076-9e80-4bf3-96a9-eea784624584","metadata":{"id":"7514e076-9e80-4bf3-96a9-eea784624584"},"source":[" &nbsp;<div style=\"text-align: right\">  \n","    &rarr; <b>NEXT: [ASPECT philosophy](./1_ASPECT_philosophy.ipynb) </b> <a href=\"\"></a> &nbsp;&nbsp;\n","     <img src=\"../assets/education-gem-notebooks_icon.png\" alt=\"icon\"  style=\"width:4%\">\n","  </div>"]}],"metadata":{"kernelspec":{"display_name":"Python 3 (ipykernel)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.5"},"colab":{"provenance":[]}},"nbformat":4,"nbformat_minor":5}
```

Comment:
<p>We probably should enforce some consistency in the bullet lists. Should they all start with a capital letter? AI says no commas "," as well.</p><p> </p><p>Add a Dependencies header for completeness and just in case we might need to add later.</p><p> </p><p>scipy - &gt; SciPy</p><p>matplotlib -&gt; Matplotlib</p><p>anlysis -&gt; analysis</p><p> </p><p>add "the" - Fundamentals of heat transport mechanisms inside the Earth</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>